testpilot/frontend/static-src/app/views/landing-page.js

```diff
@@ -42,10 +42,17 @@ export default PageView.extend({
 
   installClicked() {
     const downloadUrl = '/static/addon/addon.xpi';
+    const installButton = document.getElementsByClassName('install');
```

Comment:
nit: Should make these lookups with ampersand's `this.query()`. Even better, implement these as view bindings on a boolean property. But, I'll probably end up just merging this as-is because ampersand is annoying